base/src/main/java/uk/ac/ebi/atlas/model/experiment/baseline/BaselineExpressionPerBiologicalReplicate.java

```diff
@@ -78,4 +90,28 @@ public int hashCode() {
     public String toString() {
         return MessageFormat.format("BaselineExpressionPerBiologicalReplicate{data={0}}", data);
     }
+
+    /*
+    assumes allValues contains value
+     */
+    DOMINANCE_AMONG_RELATED_VALUES dominanceAmongRelatedValues(double[] allValuesSortedInIncreasingOrder, double value){
+        if(allValuesSortedInIncreasingOrder.length == 0 || value == 0) return DOMINANCE_AMONG_RELATED_VALUES.absent;
+        if(allValuesSortedInIncreasingOrder.length == 1) return DOMINANCE_AMONG_RELATED_VALUES.dominant;
+
+        double topValue = allValuesSortedInIncreasingOrder[allValuesSortedInIncreasingOrder.length - 1];
+        double valueFollowingTopValue = allValuesSortedInIncreasingOrder[allValuesSortedInIncreasingOrder.length - 2];
+        /*
+        Definition quoted by Nuno: a transcript is dominant if it is twice as expressed as the next value
+         */
+        if(value == topValue && value > valueFollowingTopValue * 2 ) return DOMINANCE_AMONG_RELATED_VALUES.dominant;
```

Comment:
Should be >=, also, this is simple enough but bear in mind that if blocks without braces are evil.